src/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/ValidatePackage.cs

```diff
@@ -22,7 +22,7 @@ internal static class ValidatePackage
             bool enableRuleAttributesMustMatch,
             string[]? excludeAttributesFiles,
             bool enableRuleCannotChangeParameterName,
-            string? packagePath,
+            string packagePath,
```

Comment:
This doesn't look right.

Reply:
I think this is a strict revert of the prior change.

Reply:
Sorry, this is currently blocking codeflow to installer and we don't want to delay further. It'll come back with the revert of the revert.